lib/usd/ui/CMakeLists.txt

```diff
@@ -110,6 +71,19 @@ if(IS_MACOSX OR IS_LINUX)
     mayaUsd_install_rpath(rpath ${PROJECT_NAME})
 endif()
 
+# -----------------------------------------------------------------------------
+# promoted headers
+# -----------------------------------------------------------------------------
+set(headers
+    api.h
+)
+
+mayaUsd_promoteHeaderList( 
+    HEADERS
+        ${headers}
+    BASEDIR
+        ${PROJECT_NAME}/ui
```

Comment:
Based on the commit comments and PR description, should this be `mayaUsd` instead of `${PROJECT_NAME}` since it's under the `lib/usd` directory?

Reply:
Hey @mattyjams  sorry for the misleading PR description around ${PROJECT_NAME}. I will fix the description but let me explain the problem:

The top-level cmake in super project `maya-usd/CMakeLists.txt` defines the project name
as `maya-usd`.  This is the root project. We then have bunch of mini projects underneath maya-usd project:

e,g
project(mayaUsd)
project(mayaUsdUI)
project(testMayaUsdUI)

`project` basically sets the name of the project, and stores it in the variable **PROJECT_NAME**.

Now `lib/usd/ui`  top-level cmake declares `project(mayaUsdUI)` and therefore ${PROJECT_NAME}  under this sub-directory returns `mayaUsdUI` which is a correct expected behavior.

Now, `lib/usd/schemas` is not really part of any projects and since we have restructured the sub-directories under lib, calling ${PROJECT_NAME} returns `maya-usd` from the targets that are not part of a mini projects ( e.g schemas, hdMaya, utils) . 
I have brought this up in our internal meetings and it is something to consider fixing ( something for sure for another day ). 

schema for whatever reasons needs to install it's stuff under `lib\python\mayaUsd`  and that's why ${PROJECT_NAME} would install things in  `lib\python\maya-usd` which is wrong.

I hope this makes sense.